### DISCUSSION TITLE
Display sender picture in messageListView

### DISCUSSION THREAD

**Discussion by spvince** [GetStream/stream-chat-android#2173] [General]
Hi, 
Is it possible to display the sender picture in messageListView( just beside message as it's made for receiver) ?

**Comment by adasiewiczr** [comment]
Hey @spvince currently this is not possible, but we have this on our roadmap and we'll ship it in upcoming releases

**Reply by spvince** [reply]
Hey @adasiewiczr  thanks for your answer. When this feature will be available in the futur release I would appreciate you wrote a little message in this thread.

**Reply by adasiewiczr** [reply]
Hey @spvince, yesterday we released a new version `4.17.2` that includes the requested feature. Check our [CHANGELOG](https://github.com/GetStream/stream-chat-android/blob/main/CHANGELOG.md#-added-1) for further details

**Reply by spvince** [reply]
Thanks @adasiewiczr
